OOP/OOPS_Session_1/ServiceStation.java

```diff
@@ -0,0 +1,158 @@
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.*;
+class Mechanics
+{
+	private String mechanicName,specializeCar;
+	private int rate;
+	public Mechanics(String mechanicName,String specializeCar,int rate)//Mechanics class constructor 
+	{
+		this.mechanicName=mechanicName;
+		this.specializeCar=specializeCar;
+		this.rate=rate;
+	}
+	public String getMName()//get method is used to return the value
```

Comment:
ACTION : getMName() doesn't indicate that it will return machineName. generally for getter and setter you just have to prefix variable name with set or get.